.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "makd"]
 	path = makd
 	url = https://github.com/sociomantic-tsunami/makd
+[submodule "beaver"]
+	path = beaver
+	url = https://github.com/sociomantic-tsunami/beaver.git
```

Comment:
Beaver is now at 0.3.2, I recommend using the latest version.

Reply:
There is v0.3.3 already, maybe update before merging? It only fixes a problem when trying to use DMD v2.077+ (to account for a package name change in the d-apt repo).